bundles/org.eclipse.passage.lic.api/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Automatic-Module-Name: org.eclipse.passage.lic.api
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.passage.lic.api
-Bundle-Version: 0.6.0.qualifier
```

Comment:
Just a bit confused - why the patch version has been bumped?

Reply:
You are right. I still have problems with checker configuration, to be fixed